new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -64,24 +64,24 @@ public static boolean scaleDomain(String domainUID, String clusterName, int numO
   }
 
   /**
-   * Delete the Operator release.
+   * Uninstall the Operator release.
    *
    * @param name      operator release name
    * @param namespace the name of the namespace
    * @return true on success, false otherwise
    */
 
-  public static boolean deleteOperator(String name, String namespace) {
-    return Operator.delete(name, namespace);
+  public static boolean uninstallOperator(OperatorParams params) {
+    return Operator.uninstall(params);
   }
 
   // ----------------------   domain  -----------------------------------
 
   /**
    * Create domain custom resource from the given domain yaml file.
    *
-   * @param domainUID - unique domain identifier
-   * @param namespace - name of namespace
+   * @param domainUID  - unique domain identifier
```

Comment:
You don't need the formatting here (the hyphen)... that will generated Javadoc that starts with this symbol.

Reply:
removed the formatting